fortuna/prob_model/posterior/normalizing_flow/advi/advi_approximator.py

```diff
@@ -22,10 +24,21 @@ def __init__(
             distribution is assumed to be an isotropic Gaussian, with this argument as standard deviation.
         n_loss_samples : int
             Number of samples to approximate the loss, that is the KL divergence (or the ELBO, equivalently).
+        which_params: Optional[Tuple[List, ...]]
+            Sequences of keys to the parameters of the probabilistic model for which to define the Laplace
+            approximation. If `which_params` is not available, the posterior approximation will be over all parameters.
         """
         self.std_init_params = std_init_params
         self.std_base = std_base
         self.n_loss_samples = n_loss_samples
 
+        if which_params:
+            if type(which_params) != tuple:
```

Comment:
It's better to use `isinstance` for type comparison as it can handle subclasses as well.